core/src/main/java/hudson/Util.java

```diff
@@ -1952,4 +1952,17 @@ public static String getHexOfSHA256DigestOf(byte[] input) throws IOException {
         byte[] payloadDigest = Util.getSHA256DigestOf(input);
         return (payloadDigest != null) ? Util.toHexString(payloadDigest) : null;
     }
+
+    /**
+     * Setting this flag to true enables FIPS mode
+     */
+    @SuppressFBWarnings(value = "MS_SHOULD_BE_FINAL")
+    @Restricted(NoExternalUse.class)
+    public static boolean FIPS_MODE = SystemProperties.getBoolean(Util.class.getName() + ".FIPS_MODE");
+
+    public static boolean isFipsMode() {
+        LOGGER.info("FIPS mode : " + FIPS_MODE);
```

Comment:
It looks like this method is going to be called quite often, so spamming the system log with this on `INFO` level doesn't look good.

Reply:
modified it to FINE level